src/sentry/models/group.py

```diff
@@ -563,6 +563,21 @@ def get_event_type(self):
         """
         return self.data.get("type", "default")
 
+    @property
+    def get_issue_type(self):
+        # TODO(gilbert): remove this after when type is available
+        return "error"
+
+    def get_issue_category(self):
+        # TODO(gilbert): change this impl when category is available and mappable
+        if not self.get_issue_type:
+            return None
+
+        if self.get_issue_type in frozenset(["PERFORMANCE_N_PLUS_ONE", "PERFORMANCE_SLOW_SPAN"]):
+            return "performance"
+        else:
+            return "error"
+
```

Comment:
Introduce retrieving `issue_type` and `issue_category` here. Impl is currently stubbed out until the migration for `Group` can be merged in.

Reply:
I'm handling this in https://getsentry.atlassian.net/browse/WOR-2085 WIP here: https://github.com/getsentry/sentry/pull/37889

Reply:
Nice. I'll rebase this branch once we have your stuff merged and update my code to call yours properties.

Reply:
FYI this was merged, so you can rebase now and use it.